client/src/components/InterpreterSignUpForm.js

```diff
@@ -163,7 +163,7 @@ class InterpreterSignUpForm extends Component {
                   Submit
                 </Button>
                 <Button onClick={this.handleClick} secondary>
-                  Cancel
+                  <Link to="/">Cancel</Link>
```

Comment:
You don't need to add a Link here, the redirection is already done in the handleClick function (which would be better renaming into `handleCancelClick` for clarity..